knative-sandbox-OWNERS_ALIASES

```diff
@@ -112,8 +112,13 @@ aliases:
   - devguyio
   - lionelvillard
   eventing-writers:
+  - aliok
   - devguyio
   - lionelvillard
+  - matzew
+  - odacremolbap
+  - pierDipi
+  - vaikas
```

Comment:
In this way, every approver in eventing core is an approver in repositories sponsored by Eventing which doesn't seem right.